src/commands/node/handlers.ts

```diff
@@ -171,7 +171,7 @@ export class NodeCommandHandlers implements CommandHandlers {
       this.tasks.fetchPlatformSoftware('allNodeAliases'),
       this.tasks.downloadLastState(),
       this.tasks.uploadStateToNewNode(),
-      this.tasks.setupNetworkNodes('allNodeAliases'),
+      this.tasks.setupNetworkNodes('allNodeAliases', true),
```

Comment:
```suggestion
      this.tasks.setupNetworkNodes('allNodeAliases', false),
```
don't run since network is already gone through genesis